packages/automerge-repo/src/helpers/tests/storage-adapter-tests.ts

```diff
@@ -28,185 +32,68 @@ export function runStorageAdapterTests(_setup: SetupFn, title?: string): void {
     describe("save and load", () => {
       it("should return data that was saved", async () => {
         const { adapter, teardown } = await setup()
-        await adapter.save(
-          ["storage-adapter-id"],
-          new Uint8Array([
-            56, 97, 51, 53, 99, 57, 98, 52, 45, 49, 48, 57, 101, 45, 52, 97, 55,
-            102, 45, 97, 51, 53, 101, 45, 97, 53, 52, 54, 52, 49, 50, 49, 98,
-            54, 100, 100,
-          ])
-        )
+        await adapter.save(["storage-adapter-id"], PAYLOAD_A)
 
         const actual = await adapter.load(["storage-adapter-id"])
 
-        expect(actual).toStrictEqual(
-          new Uint8Array([
-            56, 97, 51, 53, 99, 57, 98, 52, 45, 49, 48, 57, 101, 45, 52, 97, 55,
-            102, 45, 97, 51, 53, 101, 45, 97, 53, 52, 54, 52, 49, 50, 49, 98,
-            54, 100, 100,
-          ])
-        )
+        expect(actual).toStrictEqual(PAYLOAD_A)
```

Comment:
A nitpick.

In most cases, it is not recommended to share any data between the Act and Assert steps. This can lead to errors. Because any change to an input object in the Act step changes the expectations object. There are couple ways to prevent the issue: 1) hardcoding an input and expectation values; 2) using a factory function like `PAYLOAD_A()`.

Reply:
That's a good point, thanks! I've changed PAYLOAD_A etc to functions.